populus/utils/linking.py

```diff
@@ -16,11 +17,9 @@
 
 
 DEPENDENCY_RE = (
-    '__'  # Prefixed by double underscore
-    '[a-zA-Z_]'  # First letter must be alpha or underscore
-    '[a-zA-Z0-9_]{0,59}?'  # Intermediate letters
-    '_{0,59}'
-    '__'  # End with a double underscore
+    r'__'  # Prefixed by double underscore
+    r'.{36}'  # 36 characters of dubious character
```

Comment:
I have concluded that this is the only sensible way to support the new `path:sym` references.